Components/AddDestinations/AddDestinations.js

```diff
@@ -1,10 +1,36 @@
 import React, { useState, useEffect } from 'react';
-import { StyleSheet, Text, TextInput, TouchableOpacity, View, ScrollView } from 'react-native'
+import { StyleSheet, Text, TextInput, TouchableOpacity, View, ScrollView } from 'react-native';
+import { createNewDestination } from '../../apiCalls';
+import DatePicker from '../DatePicker/DatePicker';
+
 const AddDestinations = ({ navigation, route }) => {
-  // const { tripId } = route.params
+  const { tripId, handleTripsFetch } = route.params;
```

Comment:
Great work! Passing handleTripsFetch really makes the user experience better.  No matter what flow/process they use to create/update a trip it updates and renders correctly!